source/settings.txt

```diff
@@ -3,3 +3,69 @@
 ============================
 MongoDB for VS Code Settings
 ============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Configure Settings
+------------------
+
+To configure MongoDB for VS code settings:
+
+.. include:: /includes/steps/configure-settings.rst
+
+Settings
+--------
+
+The following table lists the available settings for 
+MongoDB for VS Code:
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Setting
+     - Description
+
+   * - :guilabel:`Shell`
+     - MongoDB shell to use. Defaults to ``mongo``.
+
+   * - :guilabel:`Show`
+     - If checked, shows the MongoDB view in the Visual Studio Code 
+       left navigation panel.
+
+   * - :guilabel:`Max Number of Problems`
+     - The maximum number of problems MongoDB for VS Code will generate 
+       in the Visual Studio Code :guilabel:`Problems` view.
+
+   * - :guilabel:`Confirm Run All`
+     - If checked, shows a confirmation message before running 
+       commands in a playground.
+
+   * - :guilabel:`Connection Saving: Default Connection Saving Location`
+     - Sets if and where new connections are saved. Defaults to 
+       ``Global``.
+
+   * - :guilabel:`Connection Saving: Hide Option To Choose Where To Save New Connections`
+     - If unchecked, you are prompted to choose where to save each new 
+       connection. 
+       
+       If checked, new connections are automatically saved to the 
+       :guilabel:`Default Connection Saving Location` without prompt.
+
+   * - :guilabel:`Default Limit`
+     - Number of documents to fetch when viewing documents from a 
```

Comment:
Does this affect the results of aggregations / playgrounds at all?

Reply:
It does not